Answers.md

```diff
@@ -1,9 +1,17 @@
 1. If you had to describe semantic HTML to the next cohort of students, what would you say?
+    I would tell them that there are different types of HTML markups and dividers one can use in coding HTML. However, not all HTML markups are created equal! Semantic HTML is very helpful because it gives meaning to the code and tells the browser reading your code what's going on. Semantic meaning "meaning," I'd tell the next cohort that you want to understand the semantic meaning of the HTML you're writing and use it accordingly, because it helps the next developer as well as your SEO. I'd also explain that <div> is not semantic, but I'd explain its uses within semantic HTML as a useful divider. 
 
 2. Describe some differences between ```display: block;``` and ```display: inline;```.
+    1. ```Display: block``` creates a block level element, while ```display: inline``` creates an inline-level element.
+    2.```Display: block``` defaults to the width of its container; ```Display: inline``` defaults to the width of its contents. 
+    3. ```Display: block``` doesn't allow other blocks to be displayed by it; it takes up its available space. ```Display: inline``` allows other elements to be displayed in line. This is because of the properties described in #2. 
 
 3. What are the 4 areas of the box model?
+Content - padding - border - margin 
+
 
 4. While using flexbox, what axis are you using when you use the property: ```align-items: center```?
+    Cross axis 
 
-5. What is the git command to commit staged changes as well as write a message? 
\ No newline at end of file
+5. What is the git command to commit staged changes as well as write a message? 
+    git commit -m ""
```

Comment:
Excellent comprehension answers! Love how you went into a brief yet deep dive into the differences between `display: block` and `display: inline`